include/spblas/vendor/armpl/types.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include <cstdint>
+
+namespace spblas {
+
+using index_t = std::int32_t;
```

Comment:
@chrwarm Should I be setting `index_t` to `armpl_int_t`?  Will `armpl_int_t` automatically switch between `int32_t` and `int64_t` based on whether the 64-bit or 32-bit version is linked?

Reply:
Yes, that's right

Reply:
Just updated.